### DISCUSSION TITLE
How to use the "alttab" program with i3?

### DISCUSSION THREAD

**Discussion by jidanni** [i3/i3#5886] [Q&A] [answered]
For #5883 I did
`# aptitude install alttab`
so now it is installed.

I have no idea of what to do next.

I am supposed to put something in .config/i3/config ?

https://github.com/sagb/alttab doesn't say either.

**Comment by jidanni** [comment]
OK, I commented out the line in .config/i3/config:
```
bindsym $mod+Tab workspace back_and_forth
```
and restarted i3 and did
```
$ alttab&
```
and now instead of being able to switch between *two* windows with ALT+TAB
I can only switch between *one* window. Bad luck.

**Comment by jidanni** [comment]
I see. The "alttab" program is to complicated for users like me to use.

OK, I will try something else.

**Comment by mikeredev** [comment]
Rofi offers a pretty nice "alt+tab" feature. I use this line to show my open windows/workspaces and ssh hosts: `bindsym $mod+Tab exec --no-startup-id rofi -show combi -modes combi -combi-modes window#ssh`

`bindsym $mod+comma workspace prev` and `bindsym $mod+period workspace next` will let you switch between open workspaces with the comma and full-stop keys

**Comment by jidanni** [comment]
Thank you very much, @mikeredev 
Indeed I do get a list.
However as my brain wants to use
ALT+TAB
ALT+TAB+TAB
ALT+TAB+TAB+TAB
to choose between more and less recently used windows,
checking https://github.com/davatorium/rofi/issues/38
I'm not sure if rofi can do it.

**Comment by mikeredev** [comment]
to use Alt instead of the super key you could bind it like this: `bindsym mod1+Tab exec --no-startup-id rofi -show combi -modes combi -combi-modes window` and it'd be similar (mod1 is the Alt_L key, you can check this with `xmodmap`)

**Comment by jidanni** [comment, marked as answer]
Ah. I just got alttab working, https://github.com/sagb/alttab/issues/156 , so time to bail out!
